src/lib/cli/format.ts

```diff
@@ -181,7 +181,16 @@ export class RunningSprite {
 	}
 }
 
-export function getGlyphForStatus( status: string, runningSprite: RunningSprite ): string {
+type Status =
```

Comment:
TypeScript starting to error out on my end so here's something to fix it quickly (or for good!)